crates/crates-io/lib.rs

```diff
@@ -57,6 +57,7 @@ pub struct NewCrate {
     pub repository: Option<String>,
     pub badges: BTreeMap<String, BTreeMap<String, String>>,
     pub links: Option<String>,
+    pub rust_version: Option<String>,
```

Comment:
Does crates.io need a corresponding PR to acknowledge the field in the payload?

Reply:
No, we'll just ignore it :)